src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.formbig {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.searchForm-big {
+  width: 100%;
+  height: 70px;
+}
+
+.searchForm-small {
+  width: 100%;
+  height: 42px;
+}
+
+.form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.form__label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.form-label-big:after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.form-label-small::after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%) ;
```

Comment:
have this as mentioned in figma
` box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);`

Reply:
I think what you proposed is for the hover state and when I apply what you suggested, I fail 3 tests out of 6. With mine, I only fail 1 of them :D 
but still not all six :(